### PR TITLE
#2679 - Highlight colors for Simple Objects do not match the new design

### LINKED ISSUE #2679: Highlight colors for Simple Objects do not match the new design
**Steps to Reproduce**
1. Open Ketcher
2. Add Simple objects on canvas (e.g. Shape Ellips, Shape Rectangle, Shape Line)
3. Add Benzene ring and any atoms
4. Select all and look at selection colors

**Actual behavior**
Highlight colors for Simple Objects do not match the new design

**Expected behavior**
Highlight colors for Simple Objects is match the new design

**Screenshots**
2023-05-25_14h14_44

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 112.0.5615.138 (Official Build) (64-bit)

**Ketcher version** 
[Version 2.11.0-rc.1](https://lifescience.opensource.epam.com/ketcher/index.html)

### PR DESCRIPTION
**Pull request by StarlaStarla**
Closes #2679

## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [x] unit-tests written
- [x] e2e-tests written
- [ ] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"

### CODE REVIEW

File: packages/ketcher-core/src/application/render/restruct/resimpleObject.ts [hunk 1]
```diff
@@ -189,6 +191,14 @@ class ReSimpleObject extends ReObject {
     return refPoints;
   }
 
+  getHoverPathStyle(path: any, render: Render, type: boolean) {
+    if (type) {
+      return path.attr(render.options.hoverStyle);
+    } else {
+      return path.attr({ ...render.options.hoverStyle, fill: '#fff' });
+    }
+  }
```
Comment (rrodionov91) [inline packages/ketcher-core/src/application/render/restruct/resimpleObject.ts:200]: Not sure that "type" is a good name here. Could you please change it or describe what is it?

File: packages/ketcher-core/src/application/render/restruct/resimpleObject.ts [hunk 2]
```diff
@@ -197,56 +207,61 @@ class ReSimpleObject extends ReObject {
...
+          tfx(Math.min(point[0].y, point[1].y) - scaleFactor / 8),
+          tfx(
+            Math.max(point[0].x, point[1].x) -
+              Math.min(point[0].x, point[1].x) +
+              scaleFactor / 4,
+          ),
+          tfx(
+            Math.max(point[0].y, point[1].y) -
+              Math.min(point[0].y, point[1].y) +
+              scaleFactor / 4,
```
Comment (rrodionov91) [inline packages/ketcher-core/src/application/render/restruct/resimpleObject.ts:258]: If you understood what "tfx" is please rename it

**Review by rrodionov91 — CHANGES_REQUESTED** [review]

File: packages/ketcher-core/src/application/render/restruct/resimpleObject.ts (hunk 2 above)
Comment (StarlaStarla) [reply packages/ketcher-core/src/application/render/restruct/resimpleObject.ts:258]: I found there is a separate Jira for this https://github.com/epam/ketcher/issues/2675, so I just skip it for now

Comment (StarlaStarla) [reply packages/ketcher-core/src/application/render/restruct/resimpleObject.ts:258]: I found there is a separate Jira for this https://github.com/epam/ketcher/issues/2675, so I just skip it for now

Comment (StarlaStarla) [reply packages/ketcher-core/src/application/render/restruct/resimpleObject.ts:258]: I found there is a separate Jira for this https://github.com/epam/ketcher/issues/2675, so I just skip it for now

**Comment by Zhirnoff** [comment]
Bug is fixed
2023-07-31_19h06_14

**Review by Zhirnoff — APPROVED** [review]

**Review by rrodionov91 — APPROVED** [review]

**Pull request merged**